dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Security Issue**: Adding a null check for 'registryConfig' before accessing its 'getScopeModel()' method improves the robustness of the code and prevents potential NullPointerException. This is a good practice to avoid crashes due to dereferencing null pointers. <br> **Fix**: Implement the null check as shown in the code suggestion to prevent NullPointerException. <br> **Code Suggestion**: 
 ```
 -                if (registryConfig.getScopeModel() != applicationModel) {
 +                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
 ```